Dockerfile

```diff
@@ -1,19 +1,28 @@
-ARG GO_VERSION="1.17.0"
-ARG ALPINE_VERSION="3.14"
-ARG GLOBAL_KUBE_VERSION="v1.22.1"
-ARG GLOBAL_HELM_VERSION="v3.8.1"
-ARG GLOBAL_HELM_DIFF_VERSION="v3.1.3"
-ARG GLOBAL_SOPS_VERSION="v3.7.1"
+ARG GO_VERSION="1.17.10"
```

Comment:
I've been using the latest 1.18 locally and it works fine, maybe we can update...